Code/Framework/AzCore/AzCore/Math/Quaternion.cpp

```diff
@@ -414,6 +414,75 @@ namespace AZ
         Set(x, y, z, w);
     }
 
+    Quaternion Quaternion::CreateFromEulerRadiansXYZ(const Vector3& eulerRadians)
+    {
+         const Simd::Vec3::FloatType half = Simd::Vec3::Splat(0.5f);
+        const Simd::Vec3::FloatType angles = Simd::Vec3::Mul(half, eulerRadians.GetSimdValue());
+        Simd::Vec3::FloatType sin, cos;
+        Simd::Vec3::SinCos(angles, sin, cos);
+
+        const float sx = Simd::Vec3::SelectFirst(sin);
+        const float cx = Simd::Vec3::SelectFirst(cos);
+        const float sy = Simd::Vec3::SelectSecond(sin);
+        const float cy = Simd::Vec3::SelectSecond(cos);    
+        const float sz = Simd::Vec3::SelectThird(sin);
+        const float cz = Simd::Vec3::SelectThird(cos);
+
+        // rot = rotx * roty * rotz
+        return AZ::Quaternion(
+            cx * sy * sz + sx * cy * cz,
+            cx * sy * cz - sx * cy * sz,
+            cx * cy * sz + sx * sy * cz,
+            cx * cy * cz - sx * sy * sz
+        );
+    }
+
+    Quaternion Quaternion::CreateFromEulerRadiansYXZ(const Vector3& eulerRadians)
+    {
+        const Simd::Vec3::FloatType half = Simd::Vec3::Splat(0.5f);
+        const Simd::Vec3::FloatType angles = Simd::Vec3::Mul(half, eulerRadians.GetSimdValue());
+        Simd::Vec3::FloatType sin, cos;
+        Simd::Vec3::SinCos(angles, sin, cos);
+
+        const float sy = Simd::Vec3::SelectFirst(sin);
+        const float cy = Simd::Vec3::SelectFirst(cos);
+        const float sx = Simd::Vec3::SelectSecond(sin);
+        const float cx = Simd::Vec3::SelectSecond(cos);
+        const float sz = Simd::Vec3::SelectThird(sin);
+        const float cz = Simd::Vec3::SelectThird(cos);
+
+        // rot = roty * rotx * rotz
+        return AZ::Quaternion(
+            cy * sx * cz + sy * cx * sz,
+            sy * cx * cz - cy * sx * sz,
+            cy * cx * sz - sy * sx * cz,
+            cy * cx * cz + sy * sx * sz
+        );
+    }
+    Quaternion Quaternion::CreateFromEulerRadiansZYX(const Vector3& eulerRadians)
+    {
+        const Simd::Vec3::FloatType half = Simd::Vec3::Splat(0.5f);
+        const Simd::Vec3::FloatType angles = Simd::Vec3::Mul(half, eulerRadians.GetSimdValue());
+        Simd::Vec3::FloatType sin, cos;
+        Simd::Vec3::SinCos(angles, sin, cos);
+
+        const float sx = Simd::Vec3::SelectThird(sin);
+        const float cx = Simd::Vec3::SelectThird(cos);
+
+        const float sy = Simd::Vec3::SelectSecond(sin);
+        const float cy = Simd::Vec3::SelectSecond(cos);
+        
+        const float sz = Simd::Vec3::SelectFirst(sin);
+        const float cz = Simd::Vec3::SelectFirst(cos);
```

Comment:
update: Same comment about XYZ and and First/Second/Third - also we should remove the additional line breaks to be consistent with the other two functions